### PR TITLE
Xai2

### PR DESCRIPTION
**Pull request by mshergad**
new PR

### CODE REVIEW

File: content_root/tutorial/quickstart/Fiddler_Quick_Start_Simple_Monitoring.ipynb [hunk 1]
```diff
@@ -1,656 +1,656 @@
...
+   },
+   "source": [
+    "# Fiddler Simple Monitoring Quick Start Guide\n",
+    "\n",
+    "Fiddler is the pioneer in enterprise Model Performance Management (MPM), offering a unified platform that enables Data Science, MLOps, Risk, Compliance, Analytics, and LOB teams to **monitor, explain, analyze, and improve ML deployments at enterprise scale**. \n",
+    "Obtain contextual insights at any stage of the ML lifecycle, improve predictions, increase transparency and fairness, and optimize business revenue.\n",
+    "\n",
+    "---\n",
+    "\n",
+    "You can start using Fiddler ***in minutes*** by following these five quick steps:\n",
```
Comment (DannyBrock) [inline content_root/tutorial/quickstart/Fiddler_Quick_Start_Simple_Monitoring.ipynb:16]: This says 5 steps, but we have 7 steps now.

File: .idea/fiddler-samples.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (DannyBrock) [inline .idea/fiddler-samples.iml:1]: What is this .idea directory?

File: content_root/tutorial/quickstart/Fiddler_Quick_Start_Simple_Monitoring.ipynb [hunk 3]
```diff
@@ -1,656 +1,656 @@
 {
-  "cells": [
```
Comment (DannyBrock) [inline content_root/tutorial/quickstart/Fiddler_Quick_Start_Simple_Monitoring.ipynb:2]: What is this new .ipynb_checkpoints directory?

**Comment by mshergad** [comment]
unnecessary new files

**Pull request closed without merging**